media-gfx/pencil/metadata.xml

```diff
@@ -5,4 +5,12 @@
 	<upstream>
 		<remote-id type="sourceforge">pencil-planner</remote-id>
 	</upstream>
+	<maintainer type="person">
+		<email>iba68@mail.ch</email>
```

Comment:
You need to use an e-mail address that is registered on Gentoo Bugzilla so that we will be able to assign bugs to you.